docs/content/preview/migrate/yb-voyager/performance.md

```diff
@@ -10,6 +10,109 @@ menu:
+
+It is good practice to have at least ‘C’ number of batch ingestion jobs run simultaneously, where ‘C’ is equal to the sum total of all the cores available in the entire cluster.
+
+{{< note title="Note" >}}
+
+If the cpu usage is going beyond 80%, then it is recommended to lower the parallelism a bit. Similarly if the cpu% is still low, one can increase the parallelism. Include the [–parallel-jobs](../../yb-voyager/yb-voyager-cli/#parallel-jobs) to the import data command to override the default parallelism by specifying the number of connections.
+
+{{< /note >}}
+
+- Batch size - Choosing a very small value of [--batch-size](../../yb-voyager/yb-voyager-cli/#batch-size) (Default is 100000) might result in slow performance, because the time spent on actual data import might be comparable/less to time spent on other tasks such as bookkeeping, setting up the client connection and so on.
+
+- Disk write contention - YugabyteDB servers can be passed with one or multiple disk volumes to store tablet data. If all the tablets are writing to a single disk, then write contention can slow down the ingestion speed. In such cases, configuring the [yb-tservers](../../../reference/configuration/yb-tserver/) with multiple disks helps in reducing the disk write contention; thereby increasing the throughput. Additionally, disks with higher IOPS and better throughput improve the write performance.
+
+- Number of table splits:
+
+  - For larger tables and indexes that are [hash](../../../architecture/docdb-sharding/sharding/#hash-sharding) sharded, specify the number of initial tablet splits desired as a part of the DDL statement of the table. This can be very beneficial to distribute the data of the table across multiple nodes right from the get go. Refer to [hash-sharded tables](../../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example to specify the number of tablets at table creation time.
+
+  - For larger tables and indexes that are [range](../../../architecture/docdb-sharding/sharding/#range-sharding) sharded, and the value ranges of the primary key columns are known ahead of time, pre-split them at the time of creation. Refer to [range-sharded tables](../../../architecture/docdb-sharding/tablet-splitting/#hash-sharded-tables) for an example to specify the split points.
+
+- Larger cluster - If the cluster size is increased, then the write contention is reduced and it helps with faster data ingestion.
+
+{{< note title="Note" >}}
+
+All the performance optimizations suggested for data import phase is applicable for ingestion from _files_ as well.
+
+{{< /note >}}
+
+## Experiment showcasing how to obtain faster ingestion speed
+
```

Comment:
The following migration test was run using a 40MB CSV file with 120 million rows. Data was imported using varying configurations including more parallel jobs, multiple disks, and a larger cluster.

The table schema for the test was a s follows: